lib/song_page.dart

```diff
@@ -332,119 +332,123 @@ class _MySongPageState extends State<MySongPage> {
       // @see https://api.flutter.dev/flutter/widgets/NestedScrollView-class.html
       body: OrientationBuilder(builder: (context, orientation) {
         return SafeArea(
-          child: Flex(
-            direction: orientation == Orientation.portrait
-                ? Axis.vertical
-                : Axis.horizontal,
-            children: [
-              Expanded(
-                child: NestedScrollView(
-                  controller: scrollController,
-                  headerSliverBuilder: ((context, innerBoxIsScrolled) {
-                    return [
-                      SliverOverlapAbsorber(
-                        handle: NestedScrollView.sliverOverlapAbsorberHandleFor(
-                            context),
-                        sliver: SliverAppBar(
-                          pinned: orientation == Orientation.portrait,
-                          // @see https://github.com/flutter/flutter/issues/79077#issuecomment-1226882532
-                          expandedHeight: 57,
-                          title: Text(
-                            getSongTitle(widget.songsInBook[songKey]),
-                            style: const TextStyle(fontSize: 18),
-                            maxLines: 2,
+          child: Container(
```

Comment:
Ha már itt járunk: ha ez a SafeArea kapna egy `minimum: const EdgeInsets.all(3)` beállítást, akkor (bizonyos? Android? készülékeken) a kotta nélküli versszak-szövegek nem tapadnak oda a kijelző széléhez, ergo könnyebben olvashatóak lesznek.

Reply:
Szerintem a kotta is kaphatna egy kis paddinget, jobb a szemnek, nomeg azért van még forgalomban jópár lefele hajló szélű telefon.

Reply:
Done.